bin/key_server.js

```diff
@@ -176,7 +177,7 @@ function main() {
       })
     }
   })
-  .catch(process.exit.bind(null, 8))
+  .catch(process.exit(8))
```

Comment:
Don't you want to keep the `bind`, or use a fat arrow here? I don't see how this script can work with it like this?

Reply:
@deeptibaghel does this also need to be changed?